include/z64.h

```diff
@@ -977,6 +977,80 @@ typedef struct {
     /* 0x4C */ u32 unk_4C;
 } PreRender; // size = 0x50
 
+#define TRANS_TRIGGER_OUT -20 // transition is going out
+#define TRANS_TRIGGER_OFF 0 // transition is not active
+#define TRANS_TRIGGER_IN 20 // activate transition to come in
+
+typedef enum {
+    /* 00 */ TRANS_MODE_OFF,
+    /* 01 */ TRANS_MODE_SETUP,
+    /* 02 */ TRANS_MODE_INSTANCE_INIT,
+    /* 03 */ TRANS_MODE_INSTANCE_RUNNING,
+    /* 04 */ TRANS_MODE_FILL_WHITE_INIT,
+    /* 05 */ TRANS_MODE_FILL_IN,
+    /* 06 */ TRANS_MODE_FILL_OUT,
+    /* 07 */ TRANS_MODE_FILL_BROWN_INIT,
+    /* 08 */ TRANS_MODE_08, // unused
+    /* 09 */ TRANS_MODE_09, // unused
+    /* 10 */ TRANS_MODE_INSTANT,
+    /* 11 */ TRANS_MODE_INSTANCE_WAIT,
+    /* 12 */ TRANS_MODE_SANDSTORM_INIT,
+    /* 13 */ TRANS_MODE_SANDSTORM,
+    /* 14 */ TRANS_MODE_SANDSTORM_END_INIT,
+    /* 15 */ TRANS_MODE_SANDSTORM_END,
```

Comment:
for the reason you mentioned i dont think its fair to name it after start. The sandstorm is already started in the case you mentioned so it would be more confusing